apps/ejabberd/src/mod_muc_admin.erl

```diff
@@ -163,3 +163,6 @@ data_submission() ->
                               attrs = [{<<"xmlns">>, ?NS_XDATA},
                                        {<<"type">>, <<"submit">>}]}]
           }.
+
+binary_to_bare(JID) ->
```

Comment:
I see you wrote the same function in #922. How about adding this function to the `jid` module and using the same implementation from both places. Yes, I know there is dependency between these 2 PRs because of that.